src/decoder.js

```diff
@@ -43,8 +52,9 @@ function decode({ json, complete }, token) {
   }
 }
 
-module.exports = function createDecoder(options) {
-  const { json, complete } = { json: true, ...options }
+module.exports = function createDecoder(options = {}) {
+  const json = !(options.json === false)
```

Comment:
any reason for negation compare to `false`. Just my curiosity

Reply:
eslint was complaining.